categories.js

```diff
@@ -478,7 +478,7 @@ var categories = {
       "hypochlorite",
       "bitartrate",
       "thiophosphate",
-      "crystal ",
+      "crystal ",
```

Comment:
```suggestion
      "crystal",
```

Can you revert this space? :)

Reply:
My editor must have done this by itself, I didn't touch that line and it must have slipped.